### PR TITLE
Feature/fixes and improvements

### LINKED ISSUE #77: Bug fixing & Improvements
@tokland Some notes from our last meeting + some others comments from users. Anyway, we will discuss then on Monday.
- [x] Use v25/26 API.
- [x] Review export to excel feature and fix if needed
- [x] PDF export: reduce font size
- [x] Eliminate default view as an option to a user - instead offer an option to 'reset to default settings' (under Select: add the link)
- [x] Make Organisational view the default view
- [x] Unify list and pop-up schema
- [x] Add lastUpdated, updatedBy, created, createdBy. Not sure everything can be fetched with the API. Therefore, just update what makes sense obviously
- [x] Force reload for tab when schema is changed. In particular Datasets & Programs and Indicator that are currently sharing the schema
- [x] Orgunit/SqlView Settings should be modifiable only by Admin user
- [x] Fix bug: While user is editing My Settings if he changes the key to Organisational or Default, edit panel should hide.
- [x] On popup/edit -> show hand to move rows.
- [x] Readme: Update to changes
- [] Bug: both users were looking at an organisational view which has been persisted - the order of the columns doesn't seem to be persisted? Maybe the cache? [WE COULD NOT REPRODUCE, MORE INFO NEEDED]

### PR DESCRIPTION
**Pull request by tokland**
Closes #77

### CODE REVIEW

File: js/settingDataManager.js [hunk 1]
```diff
@@ -29,6 +29,7 @@ function SettingDataPopupForm(me, settings, afterSetup)
 
 	var width = 700;
 	var height = 200;
+	var isDDAdmin = DhisUtils.idWebAppAdmin(me.user);
```
Comment (adrianq) [inline js/settingDataManager.js]: Typo. Should be isWebAppAdmin?

File: js/indexPopups.js [hunk 2]
```diff
@@ -373,6 +349,12 @@ function IndicatorPopup() {
 		table.append(me.getRowFormated("DataSets", me
 				.formatGroups(jsonData.dataSets)));
 
+		table.append(me.getRowFormated("Created By", 
+				me.formatJsonData(jsonData.user && jsonData.user.displayName)));
+
+		table.append(me.getRowFormated("Created On", $.format.date(
+				new Date(jsonData.created), "yyyy-MM-dd hh:mm a")));
```
Comment (adrianq) [inline js/indexPopups.js]: Have you thought about extracting this date format to a utils file? There are several occurrences.

File: js/indexPopups.js [hunk 3]
```diff
@@ -318,22 +302,14 @@ function IndicatorPopup() {
 	}
 
 	me.load_Data = function(id, execFunc) {
-		RESTUtil.getAsynchData(me.queryURL_Indicators + '/' + id + '.json?fields=id,code,displayName,displayShortName,description,annualized,numerator,denominator,numeratorDescription,denominatorDescription,categoryCombo[id,name],lastUpdated,indicatorGroups[id,name],indicatorType[id,name],dataSets[id,name],attributeValues[value,attribute[id,name]]',
+		RESTUtil.getAsynchData(me.queryURL_Indicators + '/' + id + '.json?fields=id,code,name,displayName,displayShortName,description,annualized,numerator,denominator,numeratorDescription,denominatorDescription,categoryCombo[id,name],lastUpdated,indicatorGroups[id,name],indicatorType[id,name],dataSets[id,name],attributeValues[value,attribute[id,name]],created,user[id,displayName]',
```
Comment (adrianq) [inline js/indexPopups.js:303]: Does name field actually exist? I thought name was changed into displayName

File: js/indexPopups.js (hunk 2 above)
Comment (adrianq) [reply js/indexPopups.js]: Actually, I have realised that you have a util method for this, right?

File: README.md [hunk 4]
```diff
@@ -11,34 +11,46 @@ DHIS2, a flexible, web-based open source information system to collect and analy
...
+
+* Check if the user role _DataDictionary admin_ exists, otherwise it will be created. By default, this role includes the authority _M_dhis-web-maintenance-appmanager_. Users with that authority are considered admins and can change some settings of the app (see below). 
+
+* Create the 2 SQL views required by the Dashboard tab.
+
+## Authorization
+
+* To use the application, a user must have at least one of the following authorities: _ALL_, _M_dhis-web-maintenance-appmanager_, _See Data Dictionary_.
+
+* To admin the application, a user must have at least one of the following authorities: _ALL_, _M_dhis-web-maintenance-appmanager_.
```
Comment (adrianq) [inline README.md:27]: I would like to understand a bit more the impact of this change. Talk later...

**Review by adrianq — CHANGES_REQUESTED** [review]
@tokland It looks really good. I did some testing as well and everything seems to be working fine. Just some minor comments and questions.

File: js/indexPopups.js (hunk 3 above)
Comment (tokland) [reply js/indexPopups.js:303]: Both fields "name" and "displayName" exist in the JSON, with the same value. So it's redundant to show both. I have not found any documentation on this, but I guess displayName should be used (and name removed).

Comment (adrianq) [reply js/indexPopups.js:303]: So... shall we delete it? Btw, drop me line when you are done with the changes. Thanks!

Comment (tokland) [reply js/indexPopups.js:303]: Yes, I'd drop usages of "name" and show only "displayName"

**Comment by tokland** [comment]
@adrianq Check the changes.

File: js/index.js [hunk 5]
```diff
@@ -1248,10 +1248,6 @@ function DataManager() {
 				"title" : "DE Groups"
 			},
 			{
-				data : 'user.displayName',
```
Comment (adrianq) [inline js/index.js]: @tokland maybe I am missing something... but why did you remove the created by field?

**Comment by adrianq** [comment]
@tokland They look good although I have some doubts with the latest commit. I added a comment, can you check it?

File: js/index.js (hunk 5 above)
Comment (tokland) [reply js/index.js]: Oops, trying to remove repeated display name, not user.display name, restored.

**Review by adrianq — APPROVED** [review]

**Comment by adrianq** [comment]
Thanks @tokland ! Seems to be working fine now...

**Pull request merged**